--- lets_encrypt.py	2020-02-11 16:24:15.732000000 -0500
+++ lets_encrypt_NEW.py	2020-02-11 16:45:02.316000000 -0500
@@ -29,9 +29,10 @@
 			print("No custom domain named {0} set for site".format(custom_domain))
 			return
 
-	click.confirm('Running this will stop the nginx service temporarily causing your sites to go offline\n'
-		'Do you want to continue?',
-		abort=True)
+	if interactive:
+		click.confirm('Running this will stop the nginx service temporarily causing your sites to go offline\n'
+			'Do you want to continue?',
+			abort=True)
 
 	if not get_config(bench_path).get("dns_multitenant"):
 		print("You cannot setup SSL without DNS Multitenancy")
